src/test/java/de/unisaarland/discanno/test/ProjectFacadeRESTTest.java

```diff
@@ -39,19 +39,15 @@ public void configureRESTFacade() {
         super.configureService(schemeRESTService);
         super.configureService(documentRESTService);
     }
+
     
-    // TODO fix this
-    // Tests have been disabled due to time reasons because the 
-    // @GeneratedValue(strategy = GenerationType.AUTO) in BaseEntity
-    // has been changed to GenerationType.IDENTITY. Persisting does not create
-    // an id while persisting
-    
-//    @Test
+    @Test
     public void testScenario1() throws JSONException, IOException, CloneNotSupportedException {
 
         Users admin = TestDataProvider.getAdmin();
         admin.setSession(BaseTest.SESSION_STR);
         em.persist(admin);
```

Comment:
It's better to create an extra method for this operation, because always calling
em.persist(...)
em.flush()
creates a lot of code duplication